app/controllers/api/autocomplete_controller.rb

```diff
@@ -33,6 +33,10 @@ def create
 
     private
 
+    def has_headers?
+      @items.file_contents.first.count == 2
```

Comment:
this would also return true if the first row of the csv is `Scooby,Do` as opposed to `Text,Value`.

maybe something like `@items.file_headings == %w[text value]` or similar if we specifically want to check the headers are in fact correct and present?

Reply:
oh yes, this is indeed an oversight! Have updated.